Formula/bind.rb

```diff
@@ -1,10 +1,17 @@
 class Bind < Formula
   desc "Implementation of the DNS protocols"
   homepage "https://www.isc.org/downloads/bind/"
-  url "https://ftp.isc.org/isc/bind/9.15.0/bind-9.15.0.tar.gz"
-  sha256 "ee9a18d8be7bcbb3928c9c96a71ea08ed045bfe57d3ef74f06f6177bbbb2560f"
+
+  # BIND releases with even minor version numbers (9.14.x, 9.16.x, etc) are
+  # stable. Odd-numbered minor versions are for testing, and can be unstable
+  # or buggy. They are not suitable for general deployment.
```

Comment:
Can we add this to `brew audit` please @Homebrew/maintainers?

Reply:
Submitted a PR for this now.